src/containers/adminDashboard/index.tsx

```diff
@@ -70,6 +70,8 @@ const AdminDashboard: React.FC<AdminDashboardProps> = ({
   const { windowType } = useWindowDimensions();
   const dispatch = useDispatch();
 
+  const [marker, setMarker] = useState<google.maps.Marker>();
```

Comment:
⛏️ Can we rename this state to something like `mapSearchMarker` so that its purpose is slightly more obvious?